assets/ui/gooeyActivateScreen.ui

```diff
@@ -17,8 +17,8 @@
           },
           {
             "type": "UIButton",
-            "text": "Select gooey for breeding",
-            "id": "breedButton"
+            "text": "Lure the gooey to follow",
```

Comment:
This is too verbose for button test. Just "Lure" would look much cleaner, and it's descriptive enough so the player won't get confused.